libs/components/datetime/src/lib/modules/date-range-picker/types/date-range-default-calculator-configs.ts

```diff
@@ -5,6 +5,31 @@ import { SkyDateRangeCalculatorType } from './date-range-calculator-type';
 import { SkyDateRangeDefaultCalculatorConfig } from './date-range-default-calculator-config';
 import { SkyDateRangeRelativeValue } from './date-range-relative-value';
 
+export const SKY_DEFAULT_CALCULATOR_IDS: SkyDateRangeCalculatorId[] = [
```

Comment:
Added this array, here, because we're referencing it multiple times in the source code.